### PR TITLE
:book: Fix broken link to docker webhook and d2iq

### PR DESCRIPTION
**Pull request by killianmuldoon**
Fix a couple of broken links in the book caught by the weekly markdown scanner.

### CODE REVIEW

**Review by killianmuldoon — COMMENTED** [review]
/area documentation

**Review by killianmuldoon — COMMENTED** [review]
/cherry-pick release-1.4

File: docs/proposals/20200423-etcd-data-disk.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ A "side effect" motivation is to open up the option of putting other directories
 
 References:
 https://docs.microsoft.com/en-us/archive/blogs/xiangwu/azure-vm-storage-performance-and-throttling-demystify
-https://archive-docs.d2iq.com/mesosphere/dcos/1.13/installing/production/system-requirements/azure-recommendations/#disk-configurations
+https://github.com/mesosphere/dcos-docs-site/blob/f15c4d9cd8c36c8a406a22b76b825b49d9fe577d/pages/mesosphere/dcos/1.13/installing/production/system-requirements/azure-recommendations/index.md?plain=1#L23C20-L23C20
```
Comment (chrischdi) [inline docs/proposals/20200423-etcd-data-disk.md:63]: optional:
```suggestion
https://github.com/mesosphere/dcos-docs-site/blob/f15c4d9cd8c36c8a406a22b76b825b49d9fe577d/pages/mesosphere/dcos/1.13/installing/production/system-requirements/azure-recommendations/index.md#disk-configurations
```

For the preview version, but that one is not great formatted either, so I'd be also okay with the code version.

**Review by chrischdi — COMMENTED** [review]
/lgtm

optional nit

**Review by chrischdi — COMMENTED** [review]
/lgtm

**Pull request merged**